pages/index.js

```diff
@@ -1,14 +1,29 @@
 import Head from 'next/head'
 import Footer from '../components/Footer'
 import Navbar from '../components/Navbar'
-import styles from '../styles/Home.module.css'
+import styles from '../styles/Home.module.css'  
+import Herosection from './herosection'
+import RecMovies from '../components/RecMovies'
+import Link from 'next/link'
 
 export default function Home() {
   return (
-    <div>
-      <h1>homepage</h1>
+  <div className="HomePage">
+    <div className="ImageSlider">
+      <Herosection/>
+    </div>
+    <div className="SelectedMovies">
```

Comment:
classNames differ from "overall" site usage of styles.
Is this a placeholder for readability or another css file used?